meimei.py

```diff
@@ -8,14 +8,33 @@
 from PIL import Image
 from dotenv import load_dotenv
 import os
+import os.path
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+from google.oauth2 import service_account
 
 load_dotenv()
 BOT_TOKEN = os.getenv("BOT_TOKEN")
 bot = commands.Bot(command_prefix="-", intents=discord.Intents.all())
 
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets"]
+credentials = service_account.Credentials.from_service_account_file("google api creds.json", scopes=SCOPES)
```

Comment:
if this bot gets put into a cloud provider we would need to separate the values out into env vars.